src/App.css

```diff
@@ -88,6 +87,46 @@ input[type="text"],input[type="password"],textarea{
     color: white;
 }
 
+.videolist-container{
+    margin-left: 6rem; 
+    padding: 1rem;
+    flex-wrap: wrap;
+}
+
+.custom-video-list{
+    margin-bottom: 3rem;
+}
+
+.videolist-container .card{
+    position: relative;
+}
+
+ .icon-trash{
+    right: 0;
+    color: white;
+    padding: 0.5rem
+}
+
+.icon-trash:hover{
+    color: black;
+    border-radius: 50%;
+
+}
+
+.videolist-container .card img{
+    width: 100px;
+    height: 100px;
```

Comment:
Try to give in rem units

Reply:
I wanted absolute size for image,so gave in px